jupyterlab/staging/package.json

```diff
@@ -126,7 +126,7 @@
     "@lumino/widgets": "^1.32.0",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "yjs": "^13.5.34"
+    "yjs": "^13.5.40"
```

Comment:
For this PR it looks like this change to `jupyterlab/staging/package.json` can be omitted.

Staging is updated automatically during a release by this script:

https://github.com/jupyterlab/jupyterlab/blob/3291708449f7fe008daf0ba1be63c65f3dbc2bb1/buildutils/src/update-core-mode.ts#L34

Undoing this change should also make the `staging-check` CI check pass.

Reply:
Thanks! I wasn't aware of that.